packages/flame/lib/src/game/flame_game.dart

```diff
@@ -17,17 +17,15 @@ import 'projector.dart';
 /// This is the recommended base class to use for most games made with Flame.
 /// It is based on the Flame Component System (also known as FCS).
 class FlameGame extends Component with Game {
+  late final CameraWrapper _cameraWrapper;
```

Comment:
Note that Flutter style guide recommends that constructors would go first in the class, before any variables are declared: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#constructors-come-first-in-a-class.
There is even a lint rule for that, though we haven't enabled it yet.

Reply:
TIL, it's different in pretty much all other OOP languages that I've programmed in. Will change when I get home!

Reply:
That is very unusual to me too, I think we can decide whether we want that rule or not separately as I imagine there are many, many violations on the entire project